UI_UX/Frontend/src/styles/AppHeader.scss

```diff
@@ -1,21 +1,81 @@
-$headerHeight: 74px;
+$headerHeight: 90px;
 
 .AppHeader {
 	height: $headerHeight;
-	background-color: #200344;
+	background-image: linear-gradient(to right, #0b3f9c, #2e58db);;
     margin: 0px 0px;
     color: white;
 
     .AppHeader_logo {
         padding: 0 25px;
         margin: auto 0;
-        height: 47px;
+        height: 73px;
     }
 
-    .AppHeader_connect{
-    	position: relative;
+    .AppHeader_exchange_rate{
     	line-height: $headerHeight;
-    	padding-left: 10px;
-    	height: max-content;
+    	font-size: 14px;
```

Comment:
As mentioned, we will eventually move to a `typography.scss`, but that can wait a bit.